src/docs/product/session-replay/getting-started.mdx

```diff
@@ -25,3 +25,28 @@ Session Replay supports all browser-based JavaScript applications. It’s built
 - [Remix](/platforms/javascript/guides/remix/session-replay/)
 - [Svelte](/platforms/javascript/guides/svelte/session-replay/)
 - [Vue](/platforms/javascript/guides/vue/session-replay/)
+
+## Backend Error Linking
+
+Backend error linking works via integrating replays with our tracing data model. When you have Sentry wired up on your frontend and backend, we have data connectivity end-to-end on your APIs and can link replays to backend endpoints.
+
+With backend error linking, the **Replay Details** view includes backend errors in the Timeline, Breadcrumbs, Errors tab, and more. You'll also see a Replays tab and a preview of a replay in the **Issue Details** page for any backend error that has a linked replay.
+
+For backend error linking to work, make sure you've set up trace propagation in your backend projects. For example:
+
+```
+Sentry.init({
+  tracePropagationTargets: ["third-party-site.com", /^https:\/\/yourserver\.io\/api/],
+});
+```
+
+Currently, backend error linking is supported for the following SDKs:
```

Comment:
Worth indicating that's the version we introduced support, any version after that included, and latest is always advised:

```suggestion
Currently, backend error linking is supported for the following SDK versions or newer:
```